span_test.go

```diff
@@ -182,6 +182,70 @@ func TestConvertSpan(t *testing.T) {
 	}
 }
 
+func TestConvertSpanWithDefault(t *testing.T) {
+	service := "my-service"
+	e := newTraceExporter(Options{
+		Service: service,
+		TraceTags: map[string]interface{}{
+			"key1": "value1",
+			"key2": false,
+			"key3": int64(2),
+		},
+	})
+
+	spanPairs := []struct {
```

Comment:
You can use the already existing span pairs, no need to add new ones.

Reply:
`newTraceExporter` needs different options and which results in differect expected results, so I added new one.
I added Options to existing spanPairs to share test patterns instead. how about this?